yetanotherco/aligned_layer
# chore: Makefile target to set up all aligned components locally

# Makefile target to set up all aligned components locally

## Description

Since there's a lot of components now that are part of Aligned, this makefile target aims to be able to set up all of them just using one command. This requires to have `tmux` installed since every different part will be hosted in a different tmux window under the aligned session.

## How to test

Just run `make setup_local_aligned_all` and do a `tmux attach`. You should see a few sessions, each one with the name of the component that is running. After a minute or so you should see all the components running including the explorer and the telemetry.

## Type of change

Please delete options that are not relevant.


## Checklist

  - [Link to the open issue addressing it]()
